aggregator/src/main/scala/com/stratio/sparkta/aggregator/Cube.scala

```diff
@@ -115,28 +96,4 @@ case class Cube(name: String,
         .map { case (name, value) => (name, operatorsMap(name).processReduce(value.map(_._2))) }
     })
   }
-
-  override def toString: String = "[Cube over " + dimensions + "]"
```

Comment:
Removed because is a Case Class. The complete toString format is (for example):
```
Cube(cubeName,List(dim1),List(Parameterizable(Map()), Parameterizable(Map(inputField -> n))),minute,10000,minute,60000)
```
It's includes the other properties on the output.